src/components/sort-by-select/SortBySelect.jsx

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next'
+
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/components/sort-by-select/SortBySelect.styles'
+import { sortByFields } from '~/constants'
+
+const SortBySelect = ({ setSortBy, sortBy }) => {
```

Comment:
Sort should be common dynamic components, and should not be aware of the items it is building. It should be mapped from incoming data.

Reply:
added getting fields for sorting from props and mapping them